### PR TITLE
EPMEDU-721: Add gradle multi-module gradle plugin

### PR DESCRIPTION
**Pull request by egorikftp**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -13,4 +13,7 @@
 /captures
 .externalNativeBuild
 .cxx
-local.properties
\ No newline at end of file
+local.properties
+awsconfiguration.json
+amplifyconfiguration.json
+/buildSrc/build
```
Comment (FlinForm) [inline .gitignore:17]: Can we remove jsons from here, if they are present?

**Review by FlinForm — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (egorikftp) [reply .gitignore:17]: done

File: buildSrc/src/main/java/com/epmedu/animeal/AnimalisPlugin.kt [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+import com.epmedu.animeal.configure.configureAndroidApplication
+import com.epmedu.animeal.configure.configureAndroidLibrary
+import com.epmedu.animeal.configure.configureBuildFeatures
+import com.epmedu.animeal.internal.applicationExtension
+import com.epmedu.animeal.internal.libraryExtension
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+import org.gradle.kotlin.dsl.create
+
+open class AnimalisPlugin : Plugin<Project> {
```
Comment (DoTheMonkeyBusiness) [inline buildSrc/src/main/java/com/epmedu/animeal/AnimalisPlugin.kt]: I guess It's better to name the plugin **AnimealPlugin**.

Comment (egorikftp) [reply buildSrc/src/main/java/com/epmedu/animeal/AnimalisPlugin.kt]: Done, please check)

**Review by DoTheMonkeyBusiness — APPROVED** [review]

**Pull request merged**